DigitalExcellence/dex-frontend
# Feature/294 implement update highlight

## Description
- Added update highlight functionality while reusing the existing highlight form
- Added pipe for formatting dates
- Renamed the `modal-highlight` component to `modal-highlight-form`

## Type of change


## Checklist


## Steps to Test or Reproduce

1. Log in as a user who has the permissions to edit the highlights
2. Go to the project overview page
3. Click the "Edit Highlight" button
4. Select the highlight for this project that you want to edit
5. Edit the information
6. Click the "Save" button
7. If the information is valid you should see a message that the highlight has been updated successfully

## Link to issue

Closes: #294
